test/browser/modal.test.jsx

```diff
@@ -0,0 +1,39 @@
+var should = require('should');
+var sinon = window.sinon;
+var React =require('react/addons');
+var TestUtils = React.addons.TestUtils;
+
+var Modal = require('../../components/modal.jsx');
+
+describe("modal", function() {
+  var modal, onClose;
+
+  function removeModal() {
+    if (modal) {
+      React.unmountComponentAtNode(modal.getDOMNode().parentNode);
+      modal = null;
+    }
+  }
+
+  beforeEach(function() {
+    onClose = sinon.spy();
+    modal = TestUtils.renderIntoDocument(
+      <Modal onClose={onClose}>
```

Comment:
We don't do anything with `onClose` yet, but we will in future tests, so I figured I might as well bust it up now.